### ISSUE TITLE
The focus indicators for tab name are confusing for ContentSwitcher

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#6837]
## Environment

> Operating system
Mac Os catalina
> Browser
Chrome: Version 84.0.4147.89 (Official Build) (64-bit)

> Assistive technology used to verify

Jaws

## Detailed description

> What version of the Carbon Design System are you using?

carbon v10 via Cloud pal
> What did you expect to happen?

Focus should be on the selceted element 
 
> What happened instead?

 focus indicators for tab name are confusing in Content Switche
> What WCAG 2.1 checkpoint does the issue violate?

Checkpoint 2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. (Level AA). The purpose of this checkpoint is to ensure that the keyboard focus indicator can be visually located so that a person knows which element has the keyboard focus. All focus indicators should be highly visible and distinguishable.(https://www.ibm.com/able/guidelines/ci162/focus_visible.html)

## Steps to reproduce the issue

1. Open Jaws
2. Open https://dev.console.test.cloud.ibm.com/security-advisor
3. Go to Configure from left nav and click alerts.
4. click on create channel
5. Click to expand (Advanced) Select alert sources and finding type
6. Use arrow key to navigate the tabs.

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/46310374/93195239-a2e71700-f766-11ea-9c89-f42def487193.png)

- Notes

**Comment by joshblack** [comment]
Hi there @reeteshsingh93! 👋 

Thanks for taking the time to make an issue. I believe this is a duplicate of: https://github.com/carbon-design-system/carbon/issues/6119 which details the problem. There is a PR over in: https://github.com/carbon-design-system/carbon/pull/6257 which should fix it, as well.

Hope this helps!
